src/Behat/Cores/Drupal8.php

```diff
@@ -230,9 +230,16 @@ public function getLastEntityId($entity_type, $bundle = NULL) {
   /**
    * {@inheritdoc}
    */
-  public function entityUri($entity_type, $entity) {
-    $path = $entity->toUrl()->getInternalPath();
-    return $path;
+  public function entityUri($entity_type, $entity, $route = 'canonical') {
+    switch($route) {
+      case 'edit':
+      case 'delete':
+      case 'add':
+        $rel .= '-form';
```

Comment:
With the latest changes, the $rel variable is not defined anymore and now it is just a concatenation.

```suggestion
        $rel = sprintf('%s-form', $route) ;
```